# Import modules

In [18]:
import pandas as pd
import os
import numpy as np

In [5]:
import re
pwd = ! pwd
pwd = re.sub('notebooks', '', pwd[0])
pwd

'/home/jovyan/work/'

# Import data

## Below are all the peak tables we prepared in the previous notebooks

In [31]:
for file in os.listdir(pwd + 'data/prepare_HCC_data/usable_peakTables'):
    
    if file.endswith('.csv') :
        
        print(file)
        print(f"shape : {pd.read_csv(pwd + 'data/prepare_HCC_data/usable_peakTables/' + file, sep=',', decimal='.', na_values=np.nan, low_memory=False).shape}")
        print()

peakTable-Agilent-186_samples.csv
shape : (189, 2160)

peakTable-Agilent_and_metadata-final-inner_merge.csv
shape : (229, 2160)

peakTable-Agilent_and_metadata-final-outer_merge.csv
shape : (937, 2160)

peakTable-Agilent_and_metadata-original-inner_merge.csv
shape : (258, 1245)

peakTable-Agilent_and_metadata-original-outer_merge.csv
shape : (934, 1245)

peakTable-Agilent_and_metadata-unfiltered-inner_merge.csv
shape : (258, 2303)

peakTable-Agilent_and_metadata-unfiltered-outer_merge.csv
shape : (934, 2303)

peakTable-metaboigniter-186_samples-remove_corr.csv
shape : (186, 1281)

peakTable-metaboigniter-186_samples.csv
shape : (186, 1501)

peakTable-metaboigniter_and_metadata-inner_merge.csv
shape : (186, 1501)

peakTable-metaboigniter_and_metadata-outer_merge.csv
shape : (934, 1501)



## For further analysis, we will use :

- ### <code>peakTable-Agilent-186_samples.csv</code> : Agilent peak table and all metadata (189 rows for 186 samples + 3 Blanks)
- ### <code>peakTable-metaboigniter-186_samples.csv</code> : metaboigniter peak table and all metadata (same 186 samples)
- ### <code>peakTable-metaboigniter-186_samples-remove_corr.csv</code> : metaboigniter peak table and all metadata (same 186 samples and strongly correlated features removed)